README.md

```diff
@@ -1,99 +1,63 @@
-[![Coverage Status](https://coveralls.io/repos/github/camsas/poseidon/badge.svg?branch=master)](https://coveralls.io/github/camsas/poseidon?branch=master)
+Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling environment that keeps on refining the dynamic placements of the workload.
 
-***Note: this repo contains an initial prototype, it may break at any time! :)***
+As we all know that as part of the Kubernetes multiple schedulers support, each new pod is typically scheduled by the default scheduler, but Kubernetes can be instructed to use another scheduler by specifying the name of another custom scheduler (Poseidon in our case) at the time of pod deployment. In this case, the default scheduler will ignore that Pod and allow Poseidon scheduler to schedule the Pod on a relevant node. We plugin Poseidon as an add-on scheduler to K8s, by using the 'schedulerName' as Poseidon in the pod template this will by-pass the default-scheduler.
 
-# Getting started
+# Key Advantages
 
-The easiest way to get Poseidon up and running is to use our Docker image:
+* Flow graph scheduling provides the following 
+  * Support for high-volume workloads placement 
+  * Complex rule constraints 
+  * Globally optimal scheduling
+  * Extremely high scalability. 
+  
+  **NOTE:** Additionally, it is also very important to highlight that Firmament scales much better than default scheduler as the number of nodes increase in a cluster.
 
-```
-$ docker pull camsas/poseidon:dev
-```
-Once the image has downloaded, you can start Poseidon as follows:
-```
-$ docker run camsas/poseidon:dev poseidon \
-    --logtostderr \
-    --kubeConfig=<path_kubeconfig_file> \
-    --firmamentAddress=<host>:<port> \
-    --statsServerAddress=<host>:<port> \
-    --kubeVersion=<Major.Minor>
-```
-Note that Poseidon will try to schedule for Kubernetes even if `kube-scheduler`
-is running. How you best avoid conflicts depends on the version of Kubernetes
-you are running:
- 1. If you are using Kubernetes <1.6, shut down `kube-scheduler` first, and make
-    sure your pods are labeled with `scheduler: poseidon`.
- 2. If you are using Kubernetes 1.6+, you do not have to to shut down
-    `kube-scheduler`, but you must specify `schedulerName: poseidon` in your pod
-    specs.
+An argument can be made from functional parity standpoint between two scheduling algorithms at this point of time. Currently, Firmament functionally is not at par with the default scheduler, especially for affinity/anti-affinity functionality as well as various other functional features currently supported in the default scheduling algorithm. It would definitely take some time to bridge the functionality gap between two scheduling algorithms. But our argument is that due to the exceptional throughput superiority of Firmament scheduler, certain use cases may not even require additional functional features such as advanced affinity/anti-affinity etc.
```

Comment:
This isn't focused and quite colloquial -- perhaps an explicit "Current Limitations" section that explains which features Poseidon lacks would be more helpful to readers.

Reply:
I just that we delete this paragraph altogether for the time being. We can provide more details, as Malte suggested, at a later time.